recipes/cfflib/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This package does not use `setuptools`, so these commands won't work due to the `--single-version-externally-managed` flag. You don't need this file or the `build.sh`, since you have the (correct) `script` entry in the meta.yaml file.